python/dask_cudf/dask_cudf/tests/test_groupby.py

```diff
@@ -53,6 +57,18 @@ def test_groupby_basic(series, aggregation):
     else:
         dd.assert_eq(a, b)
 
+@pytest.mark.parametrize("aggregation", CUMULATIVE_AGGS)
+def test_groupby_cumulative(aggregation, pdf):
```

Comment:
Should we also be testing on series groupbys here?

Reply:
So I added `Series` tests here and encountered what I think might be a bug in upstream dask. Here's a reproducer with no cuDF, modeled off of these tests:

```python
import pandas as pd
import numpy as np
import dask.dataframe as dd

np.random.seed(0)
size=10
npartitions=2


pdf = pd.DataFrame(
    {
        "xx": np.random.randint(0, 5, size=size),
        "x": np.random.normal(size=size),
        "y": np.random.normal(size=size),
    }
)

ddf = dd.from_pandas(pdf, npartitions=npartitions)

pdf_grouped = pdf.groupby('xx').xx
ddf_grouped = ddf.groupby('xx').xx

pdf_grouped.cumsum()
ddf_grouped.cumsum().compute()
```
It's a little hard for me to reason about what the result "should" be here (we're aggregating one column of a dataframe groupby and taking...the cumulative sum of that?) but the above nets me different results for the last two lines. What do you think the best thing to do here is? I could file an issue and solve it before merging this, I could xfail this, etc.

Reply:
Thanks for catching this! IMO I would add the tests and xfail, this is what I've done for other tests that would otherwise fail here due to upstream Dask issues, for example:

https://github.com/rapidsai/cudf/blob/97adac5cf92e932e4f4157f0c0c3690a5e91af09/python/dask_cudf/dask_cudf/tests/test_groupby.py#L288-L294

Reply:
raised https://github.com/dask/dask/issues/9313